text/0110-track-reform.md

```diff
@@ -60,8 +62,53 @@ export default class Foo extends LightningElement {
 
 In the example above, calling `foo()` and `bar()` will always render the latest on the next tick, while calling `baz()` might or might not update on the next tick, depending on others mutations in the current job. With this reform, they will always get the latest on the next tick, no matter what.
 
+Also, because of the way is going to be implemented, those non decorated class properties will maintain identity. Example:
+
+```js
+export default class Foo extends LightningElement {
```

Comment:
probably this example is not really needed, it is weird, and by just explaining that observability of fields is not about proxies, and we don't add the proxy there, just when you `@track`, and the identity will be preserved, should be enough